src/test/java/uk/gov/digital/ho/hocs/casework/api/CaseDataCreateCaseIntegrationTest.java

```diff
@@ -296,9 +296,9 @@ private void setupMockTeams(String caseType, int permission) throws JsonProcessi
         teamDtos.add(teamDto);
 
         mockInfoService
-                .expect(requestTo("http://localhost:8085/caseType/type/TEST"))
+                .expect(requestTo("http://localhost:8085/caseType"))
```

Comment:
Most of the real test changes are repointing wiremock to use the one remaining endpoint.